libdokan/folderlist.go

```diff
@@ -86,7 +93,15 @@ func (fl *FolderList) open(ctx context.Context, oc *openContext, path []string)
 		}
 		fl.mu.Unlock()
 
+		h, er := libkbfs.ParseTlfHandle(
+			ctx, fl.fs.config.KBPKI(), name, fl.public)
+
 		if ok {
+			if er == nil {
+				if err = fl.addToFavorite(ctx, h); err != nil {
```

Comment:
Without this, second call to `mkdir` after `rmdir` won't go through and test fails [here](https://github.com/keybase/kbfs/pull/65/files#diff-bacfce32314e3bbe1bc75470e97a335fR473). I'll add the comment in this as well as `libfuse`.